tests/scripts/quiet/quiet

```diff
@@ -1,5 +1,3 @@
-#! /usr/bin/env bash
-#
 # Copyright The Mbed TLS Contributors
```

Comment:
By our naming conventions, this file should be called `quiet.sh` (or `quiet.bash`). Not a blocker as far as I'm concerned.